src/component/Pagination.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Pagination } from 'semantic-ui-react';
+
+const PaginationComponent = ({ totalPage }) => (
```

Comment:
totalPages would be a more standardised way of naming

Reply:
got it